ironfish/src/wallet/account/encoder/bech32.ts

```diff
@@ -0,0 +1,114 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { PUBLIC_ADDRESS_LENGTH } from '@ironfish/rust-nodejs'
+import bufio from 'bufio'
+import { Bech32m } from '../../../utils'
+import {
+  AccountImport,
+  KEY_LENGTH,
+  VERSION_LENGTH,
+  VIEW_KEY_LENGTH,
+} from '../../walletdb/accountValue'
+import { AccountEncoder } from './encoder'
+
+export const BECH32_ACCOUNT_PREFIX = 'ironfishaccount00000'
+
+export class Bech32AccountEncoder implements AccountEncoder {
+  encode(value: AccountImport): string {
+    const bw = bufio.write(this.getSize(value))
+    bw.writeU16(value.version)
```

Comment:
I don't think you want to use the account version, but you want to use the encoding version which I think should be hard coded. the DB schema can change independent of the exporter changing, but the version here shouldn't be incremeneted.